src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap

```diff
@@ -841,97 +841,28 @@ hasJsx: false"
 
 exports[`Parser.getSymbols function names 1`] = `
 "functions:
```

Comment:
why are we losing the snapshot data?

Reply:
I don't know I just updated the snapshots to reflect the new test case